mempool.go

```diff
@@ -0,0 +1,84 @@
+package main
+
+import (
+	"bytes"
+	"io/ioutil"
+	"log"
+	"net/http"
+)
+
+func HandleTransaction(verb, hash string) {
```

Comment:
why is the variable called `verb`? keywords GET, POST, PATCH, etc. are usually called `method`

Reply:
True. They're less often called `verb` but `method` is more appropriate.

Reply:
oh really, I didn't know about `verb`. I've just read this [SO post](https://stackoverflow.com/questions/49907027/why-http-methods-are-also-called-verbs) stating it's not an official name though

Reply:
Okay it's an http `method` from now on!